pkg/manager/impl/execution_manager.go

```diff
@@ -1497,11 +1528,19 @@ func newExecutionSystemMetrics(scope promutils.Scope) executionSystemMetrics {
 			"delay in seconds from when an execution was requested to be created and when it actually was"),
 		PublishEventError: scope.MustNewCounter("publish_event_error",
 			"overall count of publish event errors when invoking publish()"),
+
+		WorkflowBuildSuccess: scope.MustNewCounter("build_success",
```

Comment:
workflow_build_success , as build_success could be too generic a name to understand on the dashboard.
Similar for workflow_build_failure

Reply:
done